ssh/src/main/java/ch/cyberduck/core/sftp/SFTPSession.java

```diff
@@ -278,19 +278,26 @@ private void authenticate(final SSHClient client, final Host host, final LoginCa
             final String identityAgent = new OpenSSHIdentityAgentConfigurator().getIdentityAgent(host.getHostname());
             switch(Factory.Platform.getDefault()) {
                 case windows:
```

Comment:
These checks are redundant it seems if proper checks for agent to be available in implementation for `getIdentities`